addons/zeus/functions/fnc_moduleToggleBFT.sqf

```diff
@@ -4,19 +4,26 @@ params ["_logic", "", "_activated"];
 
 if !(_activated && {local _logic}) exitWith {};
 
-private _unit = _logic getvariable ["bis_fnc_curatorAttachObject_object",objnull];
+private _unit = _logic getvariable ["bis_fnc_curatorAttachObject_object", objnull];
 deleteVehicle _logic;
 
-private _grp = group _unit;
+if (isNull _unit) then {
+    private _current = missionNamespace getVariable ["ace_map_BFT_Enabled", false];
+    ace_map_BFT_Enabled = !_current;
```

Comment:
```suggestion
    ace_map_BFT_Enabled = !_current;
    publicVariable "ace_map_BFT_Enabled";
```

is this safe? will it stay that way when someone fiddles with their local cba settings afterward? :thinking:

Reply:
Gute frage, sollte eigentlich nicht.